analyze_data.py

```diff
@@ -100,9 +112,12 @@ def save_to_file(out_folder, animal_name, dataframe):
     Returns:
         None:
     """
+    file_name = "%s.csv" % (animal_name)
```

Comment:
This file_name doesn't include the output folder.  It will place the file wherever you are calling this script from instead.

DataFrame is the static DataFrame definition, not dataframe which is an instance of DataFrame that we are passing into this function.